config/locales/en.yml

```diff
@@ -14,15 +14,16 @@ en:
 
   spree:
     slides: "Slides"
-    all_slides: 'All Slides'
-    slide_locations: 'Slide Locations'
-
+    spree_slider:
+      config_name: 'Spree Slider'
+    spree_slider_locations:
+      config_name: 'Spree Slider Locations'
+      
   activerecord:
     models:
       spree/slide: 'Slide'
 
   spree_slider:
-    config_name: 'Spree Slider'
     config_description: 'Manage Spree Slider contents'
```

Comment:
can we pull of this change without removing these changes..it is hard to follow up what is happening here?